src/seeders/20210114194825-file-statuses.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
```

Comment:
I _think_ I'd rather file status be a field on the `Files` table, hopefully as an enum.

Reply:
Sounds good.